### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by ghost** [bassamanator/everything-sovol-sv06#4] [Q&A] [answered]
While searching for info about the SV06, I often find suggestions to visit the Sovol discord server, but the only invite link I was able to find, has expired. Some comments suggested that the server was basically inactive anyway, so perhaps it's gone now? Does the server still exist? I'd really like an invite if anyone would be so kind :)

I also gathered that there's a facebook group, but I don't use FB, so that's why I was keen on discord.

Thanks!

**Comment by bassamanator** [comment]
As far as I know, they don't have a discord server. Would certainly be useful to get live help!

**Comment by armishmajerj** [comment, marked as answer]
Here is the link I have from their server. Not entirely sure if it's owned/operated by them or just someone else started one for their printers. https://discord.gg/zzu564CKCe

**Reply by ghost** [reply]
Thanks mate! Even if this is an unofficial server, a helpful community like this is very valuable. Cheers!
